barretenberg/docs/versioned_docs/version-v3.0.0-nightly.20251212/how_to_guides/how-to-solidity-verifier.md

```diff
@@ -60,7 +60,7 @@ You'll likely see a warning advising you to not trust pasted code. While it is a
 
 :::
 
-To compile our the verifier, we can navigate to the compilation tab:
+To compile the verifier, we can navigate to the compilation tab:
```

Comment:
Note from Adam: nightly will just be updated automatically from basic version anyway so no need to update here